Quaver.Shared/Screens/Gameplay/GameplayScreen.cs

```diff
@@ -493,6 +493,7 @@ private void HandlePauseInput(GameTime gameTime)
                     return;
 
                 // Properly fade in now.
+                // TODO: why is this here instead of wherever the restart key press is handled?
```

Comment:
From what I can see it's handling the specific case of letting go of the pause key before it fully fades in. I suppose you could move it, but it might have an unintended result because the restart and pause are treated as separate actions that control the transition. Up to you if you want to refactor this further.